src/lib/components/topbar/topbar.scss

```
@@ -5,10 +5,15 @@
   background-color: $action-lighten-5;
   box-sizing: border-box;
   display: flex;
-  justify-content: flex-end;
+  justify-content: space-between;
   padding: $space-16 $space-32;
   width: 100%;
 
+  &__topright-container {
```

O nome não pode referenciar comportamento, e sim dizer contexto.

```suggestion
  &__options {
```